test/testDrift.js

```diff
@@ -137,5 +137,20 @@ describe("Drift", () => {
         expect(drift.trigger.enabled).toBe(false);
       });
     });
+
+    describe("#destroy", () => {
+      it("should hide and unbind events", function() {
+        const anchor = document.querySelector(".test-anchor");
+        const drift = new Drift(anchor);
+
+        const hideSpy = spyOn(drift.trigger, "_hide");
```

Comment:
Looks like eslint doesn't recognize `spyOn` here, causing the tests to break. A quick search online shows that we should add (I assume) Jasmine to `.eslintrc.json` to fix this.
https://github.com/eslint/eslint/issues/1176#issuecomment-342004125

[travis results](https://travis-ci.org/imgix/drift/jobs/563340394) for reference

Reply:
Looks like all we actually needed was to add `spyOn` as an eslint global.